certbot/plugins/standalone_test.py

```diff
@@ -21,8 +25,8 @@ class ServerManagerTest(unittest.TestCase):
 
     def setUp(self):
         from certbot.plugins.standalone import ServerManager
-        self.certs = {}
-        self.http_01_resources = {}
+        self.certs = {}  # type: Dict[bytes, Tuple[OpenSSL.crypto.PKey, OpenSSL.crypto.PKey]]
```

Comment:
I think the last type in the tuple is `OpenSSL.crypto.X509`.

Reply:
yes, fixed